dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/plugins/OidcBaseClientValidator.java

```diff
@@ -92,10 +95,45 @@ public OidcBaseClient validateCreateUpdate() throws OidcServerException {
         return this.client;
     }
 
+    /**
+     * check for disallowed characters that could allow javascript to be fed back to ui.
+     */
+    private void detectIllegalChars() throws OidcServerException {
+        detectCh(client.getClientId());
+        detectCh(client.getClientSecret());
+        detectCh(client.getRedirectUris());
+        detectCh(client.getClientName());
+        detectCh(client.getPostLogoutRedirectUris());
+        detectCh(client.getPreAuthorizedScope());
+        detectCh(client.getFunctionalUserId());
+        detectCh(client.getFunctionalUserGroupIds());
+    }
+
+    // detect illegal chars
+    private void detectCh(@Sensitive String s) throws OidcServerException {
```

Comment:
The method should just be named `detectIllegalChars()`.

Reply:
seemed wordy but ok